Automattic/woocommerce-payments
# Prefill Stripe KYC data for test accounts with hardcoded data

#### Description
Fixes server issue 2935. 
No tests were provided due to the absence of the test for the file.

#### Changes proposed in this Pull Request

Prefill Stripe KYC with hard-coded data such as:
- Fake name: "John Woolliams"
- Address: "1040 Grand Ave, South San Francisco, California(CA), 94080" (randomly generated with [this](https://www.bestrandoms.com/random-address))
- Country: United States
- Phone: +10000000000
- SSN last 4: 0000
- Date of birth: 1 Jan 1980
- MCC: Software (5734)
- URL: http://wcpay.test

#### Testing instructions

* Checkout to branch `add/2935-prefill-test-mode-stripe-kyc`
* Remove the existing WCPay account from the server database if you have one locally. Clear the cache
* Go to Payments->Connect page and proceed with onboarding the new WCPay account
* During the mode choice, pick `Set up test payments`
    ![image](https://user-images.githubusercontent.com/79862886/231492721-60545771-58e9-40d7-93ee-69edd55c9e7c.png)
* Ensure that name, country are prefilled and phone, address, dob, ssn_last_4, MCC, url are not asked during the KYC
    <img width="560" alt="image" src="https://user-images.githubusercontent.com/79862886/231699347-bbc564b5-e522-43dd-a31c-b23b5a774e5e.png">
* Input email and then use test account for payouts (next form)
    <img width="540" alt="image" src="https://user-images.githubusercontent.com/79862886/231699798-379a8aa6-e3f1-451a-b520-837be482406d.png">
* Check that all the data is valid before accepting ToS.
    ![image](https://user-images.githubusercontent.com/79862886/231700134-27f52222-f775-4811-b073-64e16c27b5a4.png)
* Verify your account is onboarded and complete.
    ![image](https://user-images.githubusercontent.com/79862886/231493904-371341fb-2c5b-43c1-a714-30f70eae6c57.png)


-------------------


**Post merge**
